amqp/src/main/scala/akka/stream/contrib/amqp/model.scala

```diff
@@ -4,6 +4,7 @@
 package akka.stream.contrib.amqp
 
 import scala.collection.immutable.Seq
```

Comment:
I prefer that we are explicit over having hidden magic, especially since main purpose of these sources is to act as good samples (that will be copy pasted)